docker-compose.prod.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.0-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=people_depot
+      - POSTGRES_PASSWORD=people_depot
+      - POSTGRES_DB=people_depot_dev
+    container_name: people_depot_db
```

Comment:
Setting the container name might not be a good idea. I'll have to look up the reasoning for this again.

Reply:
Oh yeah, the container_name allows you to refer to it by name in a `docker` command. If you intend to refer to it using docker-compose, like `docker-compose run db...` then you only need to use the service name. Setting the container name also restricts docker-compose from generating multiple instances of it. I'm not sure if there's an advantage to that or not. So the reasoning for not setting it is really that it's not very useful.